client/me/help/help-contact-form/index.jsx

```diff
@@ -175,6 +197,22 @@ export class HelpContactForm extends React.PureComponent {
 
 	getSibylQuery = () => ( this.state.subject + ' ' + this.state.message ).trim();
 
+	doRequestSite = () => {
+		if ( resemblesUrl( this.state.userDeclaredUrl ) ) {
+			// The API would reject something like "https://wp.com/slug".
+			// It'd need to either be "http(s)://wp.com" or "wp.com".
+			const url = this.state.userDeclaredUrl;
+			const query = url.includes( '://' )
+				? new URL( this.state.userDeclaredUrl ).hostname
+				: new URL( 'http://' + this.state.userDeclaredUrl ).hostname;
+
+			this.props
+				.requestSite( query )
+				.then( ( siteData ) => this.setState( { siteData, errorData: '' } ) )
+				.catch( ( error ) => this.setState( { errorData: error.error, siteData: '' } ) );
```

Comment:
It's more common to use `null` instead of empty string for cases like this. Especially for `siteData` which is an object, not a string.

Reply:
Thanks, fixed!